main.cpp

```diff
@@ -108,8 +108,54 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+    struct Steps
+    {
+        void stepForward();
+        int stepSize();
+    };
+
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+    Steps leftFoot;
+    Steps rightFoot;
```

Comment:
`leftFoot` and `rightFoot` are not instances of `Steps`.
They are instances of `Foot`.

Name things what they are.
- [ ] revise